apps/charterafrica/src/payload/fields/ecosystem/socialMediaFields.js

```diff
@@ -0,0 +1,85 @@
+import airtableColumnSelect from "./airtableColumnSelect";
+
+function socialMediaColumns({ tableField, schema }) {
+  return {
+    label: "Social Media",
+    type: "group",
+    name: "socialMediaColumns",
+    fields: [
+      airtableColumnSelect({
+        schema,
+        tableField,
+        overrides: {
+          name: "twitter",
+          label: "Twitter",
+        },
+      }),
+      airtableColumnSelect({
+        schema,
+        tableField,
+        overrides: {
+          name: "tiktok",
+          label: "Tiktok",
+        },
+      }),
+      airtableColumnSelect({
+        schema,
+        tableField,
+        overrides: {
+          name: "facebook",
+          label: "Facebook",
+        },
+      }),
+      airtableColumnSelect({
+        schema,
+        tableField,
+        overrides: {
+          name: "linkedIn",
+          label: "Linkedin",
+        },
+      }),
+      airtableColumnSelect({
+        schema,
+        tableField,
+        overrides: {
+          name: "instagram",
+          label: "Instagram",
+        },
+      }),
+      airtableColumnSelect({
+        schema,
+        tableField,
+        overrides: {
+          name: "youtube",
+          label: "Youtube",
+        },
+      }),
+      airtableColumnSelect({
+        schema,
+        tableField,
+        overrides: {
+          name: "telegram",
+          label: "Telegram",
+        },
+      }),
+      airtableColumnSelect({
+        schema,
+        tableField,
+        overrides: {
+          name: "discord",
+          label: "Discord",
+        },
+      }),
+      airtableColumnSelect({
+        schema,
+        tableField,
+        overrides: {
+          name: "whatsapp",
+          label: "Whatsapp",
+        },
+      }),
+    ],
```

Comment:
```suggestion
    ],
    admin: {
      hideGutter: true,
    },
```

We need to review how the CMS _looks_ as well.
<img width="719" alt="Screenshot 2023-09-12 at 09 35 16" src="https://github.com/CodeForAfrica/ui/assets/1779590/87b71f08-4626-450c-967e-f2d2d44b4ca8">